test/src/main/java/io/strimzi/test/TestUtils.java

```diff
@@ -393,6 +393,26 @@ public static String changeRoleBindingSubject(File roleBindingFile, String names
         }
     }
 
+    public static String changeDeploymentNamespaceUpgrade(File deploymentFile, String namespace) {
+        YAMLMapper mapper = new YAMLMapper();
+        try {
+            JsonNode node = mapper.readTree(deploymentFile);
+            // Change the docker org of the images in the 04-deployment.yaml
+            ObjectNode containerNode = (ObjectNode) node.get("spec").get("template").get("spec").get("containers").get(0);
```

Comment:
I guess you can simplify this by following code `node.at("/spec/template/spec/containers")`, so no need to call method `.get()` many times